### PR TITLE
authz: code clean up and remove unused code

### PR DESCRIPTION
**Pull request by yangminzhu**
Some clean up and remove unused code.

### CODE REVIEW

File: pilot/pkg/model/authorization.go [hunk 1]
```diff
@@ -26,44 +26,49 @@ var (
...
 
 	return policy, nil
 }
 
-// ListAuthorizationPolicies returns the AuthorizationPolicy for the workload in root namespace and the config namespace.
-// The first one in the returned tuple is the deny policies and the second one is the allow policies.
-func (policy *AuthorizationPolicies) ListAuthorizationPolicies(configNamespace string, workloadLabels labels.Collection) (
-	denyPolicies []AuthorizationPolicyConfig, allowPolicies []AuthorizationPolicyConfig) {
+// ListAuthorizationPolicies returns the deny and allow AuthorizationPolicy for the workload in the given namespace.
+func (policy *AuthorizationPolicies) ListAuthorizationPolicies(namespace string, workload labels.Collection) (
```
Comment (incfly) [inline pilot/pkg/model/authorization.go:70]: why rename workloadLabels to workload? old var name seems accurate.

Comment (yangminzhu) [reply pilot/pkg/model/authorization.go:70]: to make it shorter, the `labels.Collection` should be enough to tell it's labels.

**Pull request merged**